index.js

```diff
@@ -88,21 +89,17 @@ client.on("message", (msg) => {
 		return;
 	}
 
-	if (c.includes("gourd")) {
+	if (c.match(/\bgourd\b/)) {
 		msg.channel.send("I have a " + randomString(responses.gourdList));
 		return;
 	}
 
-	if (
-		(c.includes("friends") || c.includes("friend")) &&
-		(c.includes("message") || c.includes("messages")) &&
-		msgAuthorId === KristenUserId
-	) {
+	if (c.match(/\bfriends* messages*\b/) && msgAuthorId === KristenUserId) {
```

Comment:
Would the following phrase trigger the bot?

`Haku, show me messages from my friends!`

Reply:
no it won't.
need to fix this